task/buildah/0.1/buildah.yaml

```diff
@@ -207,6 +213,13 @@ spec:
       [ -n "$COMMIT_SHA" ] && LABELS+=("--label" "vcs-ref=$COMMIT_SHA")
       [ -n "$IMAGE_EXPIRES_AFTER" ] && LABELS+=("--label" "quay.expires-after=$IMAGE_EXPIRES_AFTER")
 
+      ENTITLEMENT_PATH="/entitlement"
+      if [ -n "${ENTITLEMENT_SECRET}" ] && [ -d "$ENTITLEMENT_PATH" ]; then
+        cp -r /entitlement /tmp/entitlement
+        VOLUME_MOUNTS="${VOLUME_MOUNTS} --volume /tmp/entitlement:/etc/pki/entitlement"
+        echo "Adding the entitlement to the build"
```

Comment:
nitpick: when printing the log, it already added the `entitlement` .. will `Added the entitlement to the build` sound good here?

Reply:
Hmm, as I understand it at this point nothing was added, only variable was changed. The mounting to the buildah build command happens below

```
unshare -Uf $UNSHARE_ARGS --keep-caps -r --map-users 1,1,65536 --map-groups 1,1,65536 -- buildah build \
        $VOLUME_MOUNTS \
```